testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/KeycloakRealmImportOperatorProvisionerTest.java

```diff
@@ -0,0 +1,373 @@
+import io.fabric8.kubernetes.client.dsl.NonNamespaceOperation;
+import io.fabric8.kubernetes.client.dsl.Resource;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Test the Keycloak Operator provisioning model and APIs
+ *
+ * See <br>
+ *     - https://github.com/keycloak/keycloak-operator/tree/master/deploy/examples
+ * <p>
+ * Not all examples are actually run, but these tests are more about the framework functionality verification than
+ * about the Keycloak operator testing.
+ */
+@Slf4j
+@CleanBeforeAll
+//@Disabled("WIP - Disabled until global-test.properties is configured with the required property")
+public class KeycloakRealmImportOperatorProvisionerTest {
+	private static KeycloakRealmImportOperatorProvisioner KEYCLOAK_OPERATOR_PROVISIONER;
+
+	private static final String POSTGRESQL_NAME = "postgresql";
+	private static final String POSTGRESQL_DATABASE = "keycloak";
+	private static final String POSTGRESQL_PASSWORD = "pippobaudo1234";
+	private static final String POSTGRESQL_USER = "user09M";
+
+	private static final PostgreSQLImageOpenShiftApplication pgSQLApplication = new PostgreSQLImageOpenShiftApplication() {
+		@Override
+		public String getName() {
+			return POSTGRESQL_NAME;
+		}
+
+		@Override
+		public String getUser() {
+			return POSTGRESQL_USER;
+		}
+
+		@Override
+		public String getPassword() {
+			return POSTGRESQL_PASSWORD;
+		}
+
+		@Override
+		public String getDbName() {
+			return POSTGRESQL_DATABASE;
+		}
+	};
+	private static final PostgreSQLImageOpenShiftProvisioner POSTGRESQL_IMAGE_PROVISIONER = new PostgreSQLImageOpenShiftProvisioner(
+			pgSQLApplication);
+
+	private static KeycloakRealmImportOperatorProvisioner initializeOperatorProvisioner(final Keycloak keycloak,
+			final String appName) {
+		KeycloakRealmImportOperatorProvisioner operatorProvisioner = new KeycloakRealmImportOperatorProvisioner(
+				new KeycloakRealmImportOperatorApplication() {
+
+					@Override
+					public Keycloak getKeycloak() {
+						return keycloak;
+					}
+
+					@Override
+					public String getName() {
+						return appName;
+					}
+				});
+		return operatorProvisioner;
+	}
+
+	private String name;
+	private String realmName;
+
+	private static final Map<String, String> matchLabels = new HashMap<>();
+
+	@BeforeAll
+	public static void createOperatorGroup() throws IOException {
+		matchLabels.put("app", "sso");
+		IntersmashExtension.operatorCleanup();
+		// create operator group - this should be done by InteropExtension
+		OpenShifts.adminBinary().execute("apply", "-f", OperatorGroup.SINGLE_NAMESPACE.save().getAbsolutePath());
+	}
+
+	@BeforeEach
+	public void cleanup() throws IOException {
+		if (!Objects.isNull(KEYCLOAK_OPERATOR_PROVISIONER))
+			KEYCLOAK_OPERATOR_PROVISIONER.unsubscribe();
+	}
+
+	@AfterAll
+	public static void removeOperatorGroup() {
+		OpenShifts.adminBinary().execute("delete", "operatorgroup", "--all");
+		if (!Objects.isNull(KEYCLOAK_OPERATOR_PROVISIONER))
```

Comment:
let's remove the condition and always execute `dismiss` as the last step.

Reply:
we cannot: since the provisioner is initialized when executing the single tests (e.g. in `public void exampleSso() {`) and we would have a null pointer exception here in case something goes wrong before the test is started

Reply:
why is it broken? it just starts and ends inside a single test method instead of spanning across multiple tests with the confusing dummy initialization